README.md

```diff
@@ -52,14 +52,53 @@ Then grab and build the `leapchat` source:
 go get github.com/cryptag/leapchat
 ```
 
+### JavaScript and Node Setup
+
+Install Node v14. We recommend using the [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm) package
+to manage your node environments.
+
+If you're using NVM, you can install the correct node version by running:
+
+```
+$ nvm install   # run from inside of leapchat/ dir, uses .nvmrc file
+$ nvm install v14.0.0   # run from anywhere
+```
+
+Then, to configure the use of the correct node version whenever you enter the project:
+
+```
+$ cd ~/code/leapchat && nvm use
+```
+
+To install JavaScript dependencies:
+
+```
+$ npm install
+```
+
+In development, when you want to see your frontend code changes immediately on a browser refresh, run
+the command that boots up a watch process to re-compile the frontend whenever a file changes:
+
+```
+$ npm run dev
+```
+
+In order to do a one-time build of the production assets:
+
+```
+$ npm run build
+```
+
+The frontend is served through an HTTP server running in the go binary. This allows us to make API requests
+from the browser without any CORS configuration.
 
 ### macOS Instructions
 
 If you don't already have Postgres 9.5 to Postgres 12 installed and
 running, install it with Homebrew:
 
 ```
-brew install postgresql@10
+brew install postgresql@12
```

Comment:
`$ ` prefix?

Reply:
Or we remove the `$`s so it's easy to copy and paste the lines into their shall; that's what I've moved to even though that indeed makes it less clear that these are commands.